src/_tutorials/language/futures.md

```diff
@@ -269,9 +282,11 @@ https://gist.github.com/chalin/5e7eb16cd725f0a51926b85e3caa64b5
 
 import 'dart:async';
 
-printDailyNewsDigest() {
+Future<void> printDailyNewsDigest() {
   final future = gatherNewsReports();
-  future.then((news) => print(news));
+  return future.then((news) => print(news));
+  // You don't *have to* return the future here. But if you don't, callers can't
+  // await it.
```

Comment:
Except that the way you can call them will be different...

Reply:
Hmm, I thought the way you call them will stay the same?

It will maintain similarity behind how they look, but not in how they run. If the goal is to show what async/await translates to in terms of the Future API, its best to include the fact that it returns IMO. It might be less to throw at the user to elide it, but I'm not sure it will help to show them less in this case?

Reply:
I think we want to show Future code that's equivalent to the async-await version of the code, in terms of what gets executed when.